### PR TITLE
Docker api

### PR DESCRIPTION
**Pull request by arckosfr**
Amélioration de l'API

### CODE REVIEW

File: libs/docker.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ var docker = new Docker({ socketPath: socket });
 
 docker.list = function(cb) {
 	docker.listContainers({all: true}, function(err,containers){
-			cb(containers);
+      console.log(containers.length);
```
Comment (Hydrog3n) [inline libs/docker.js]: Peu être qui faut pas laisser ça :D

Comment (arckosfr) [inline libs/docker.js]: oups :D

**Comment by arckosfr** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/saedbox/Saedbox/pull-request/10/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=saedbox/Saedbox)

**Comment by farfeduc** [comment]
:+1:

**Pull request merged**